.github/ISSUE_TEMPLATE/card_author_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Card Author Bug Report
+about: Create a bug report for problems authoring Adaptive Crds
```

Comment:
Crds -> Cards

Reply:
I don't believe people will understand the distinction between "rendering" and "authoring" Adaptive Cards. IMHO we should phrase things differently:
- Create a bug report for a problem related to rendering an Adaptive Cards in your own application
- Create a bug report for a problem related to an Adaptive Card not displaying or working properly in a host such as Outlook or Teams

Reply:
- Good catch on the typo there...my laptop 'a' key seems to not be happy
- Good idea...I've wordsmith'd the name and about sections in each bug to add some clarity on when each bug template should be used